Python/Product/TestAdapter.Executor/Utils/TestUtils.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Python Tools for Visual Studio
+// Copyright(c) Microsoft Corporation
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the License); you may not use
+// this file except in compliance with the License. You may obtain a copy of the
+// License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// THIS CODE IS PROVIDED ON AN  *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+// OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION ANY
+// IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR PURPOSE,
+// MERCHANTABILITY OR NON-INFRINGEMENT.
+//
+// See the Apache Version 2.0 License for specific language governing
+// permissions and limitations under the License.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.PythonTools.TestAdapter.Utils {
+    static class TestUtils {
+
+        // For a small set of tests, we'll pass them on the command
+        // line. Once we exceed a certain (arbitrary) number, create
+        // a test list on disk so that we do not overflow the 
+        // 32K argument limit.
+        internal static string CreateTestList(IEnumerable<string> tests) {
```

Comment:
`testList` sounds like a list, not a file path. I see you used `testListFilePath` elsewhere, which is what I'd do here too.

And you can implement it this way:
```
var testListFilePath = Path.GetTempFileName();
File.WriteAllLines(testListFilePath, tests, new UTF8Encoding(false));
return testListFilePath;
```

Reply:
Also, I see there's another `CreateTestList` somewhere else. It would be good to make it use this one.

Reply:
yup i updated the old location

Reply:
TestExecutorUnitTest.cs